types/jsnox/jsnox-tests.ts

```diff
@@ -8,7 +9,7 @@ interface PersonProps {
     age: number;
 }
 
-const Person: React.ClassicComponentClass<PersonProps> = React.createClass<PersonProps, {}>({
+const Person: React.ClassicComponentClass<PersonProps> = createReactClass<PersonProps, {}>({
```

Comment:
I don't think there's any need to involve `create-react-class` here? It's better if types have fewer dependencies on each other.

Reply:
I was trying to avoid changing the intent of the tests in dependent packages. I'm not familiar with `jsnox` (or most of these other libraries) so I wasn't sure if the `createClass` usage here was deliberate. Would you prefer to see instances like this changed to use component classes to avoid inter-dependencies?

Reply:
If it's possible, it would be better to have a pure-`react` test. Thanks.